src/OrchardCore.Modules/OrchardCore.Templates/Views/_ViewImports.cshtml

```diff
@@ -1,4 +1,6 @@
 @inherits OrchardCore.DisplayManagement.Razor.RazorPage<TModel>
+@using Microsoft.Extensions.Localization;
```

Comment:
Why the usings are here?!! only one view use them

Reply:
So people don't have to wonder why T.Plural doesn't work if they use it in another view.

Reply:
This looks fine to me, no big deal.